src/api/status/src/js/queue-stats.js

```diff
@@ -0,0 +1,12 @@
+const { fetch } = require('@senecacdot/satellite');
+
+module.exports = async function getJobCount() {
+  try {
+    const data = await fetch(`${process.env.POSTS_URL}/feeds/info`, { method: 'GET' });
+    const { queueInfo } = await data.json();
+    return queueInfo.jobCnt;
+  } catch (error) {
+    console.error(error);
```

Comment:
Use the `logger` in Satellite, see https://github.com/Seneca-CDOT/satellite#logger:

```js
logger.warn({ error }, 'message here...');
```